src/tools.rs

```diff
@@ -580,11 +580,7 @@ impl rusqlite::types::ToSql for EmailAddress {
 
 /// Makes sure that a user input that is not supposed to contain newlines does not contain newlines.
 pub(crate) fn improve_single_line_input(input: &str) -> String {
-    input
-        .replace('\n', " ")
-        .replace('\r', " ")
-        .trim()
-        .to_string()
+    input.replace(['\n', '\r'], " ").trim().to_string()
```

Comment:
```suggestion
    input.replace(['\n', '\r'], ' ').trim().to_string()
```

this can probably be a character literal as well.

Reply:
No, `replace` wants a `&str` there.

Reply:
i see, nm :)